pacai/ui/captureGraphicsDisplay.py

```diff
@@ -1,101 +1,29 @@
+from pacai.ui.pacmanDisplay import AbstractPacmanGraphics
 
-DEFAULT_GRID_SIZE = 30.0
-INFO_PANE_HEIGHT = 35
-BACKGROUND_COLOR = graphicsUtils.formatColor(0, 0, 0)
-WALL_COLOR = graphicsUtils.formatColor(0.0 / 255.0, 51.0 / 255.0, 255.0 / 255.0)
-INFO_PANE_COLOR = graphicsUtils.formatColor(0.4, 0.4, 0)
-SCORE_COLOR = graphicsUtils.formatColor(.9, .9, .9)
-PACMAN_OUTLINE_WIDTH = 2
-PACMAN_CAPTURE_OUTLINE_WIDTH = 4
-
-GHOST_COLORS = [
-    graphicsUtils.formatColor(0.9, 0, 0),  # Red
-    graphicsUtils.formatColor(0, 0.3, 0.9),  # Blue
-    graphicsUtils.formatColor(0.98, 0.41, 0.07),  # Orange
-    graphicsUtils.formatColor(0.1, 0.75, 0.7),  # Green
-    graphicsUtils.formatColor(1.0, 0.6, 0.0),  # Yellow
-    graphicsUtils.formatColor(0.4, 0.13, 0.91),  # Purple
-]
-
-TEAM_COLORS = GHOST_COLORS[:2]
-
-GHOST_SHAPE = [
-    (0, 0.3),
-    (0.25, 0.75),
-    (0.5, 0.3),
-    (0.75, 0.75),
-    (0.75, -0.5),
-    (0.5, -0.75),
-    (-0.5, -0.75),
-    (-0.75, -0.5),
-    (-0.75, 0.75),
-    (-0.5, 0.3),
-    (-0.25, 0.75)
-]
-GHOST_SIZE = 0.65
-SCARED_COLOR = graphicsUtils.formatColor(1, 1, 1)
-
-GHOST_VEC_COLORS = list(map(graphicsUtils.colorToVector, GHOST_COLORS))
-
-PACMAN_COLOR = graphicsUtils.formatColor(255.0 / 255.0, 255.0 / 255.0, 61.0 / 255)
-PACMAN_SCALE = 0.5
-
-# Food
-FOOD_COLOR = graphicsUtils.formatColor(1, 1, 1)
-FOOD_SIZE = 0.1
-
-# Laser
-LASER_COLOR = graphicsUtils.formatColor(1, 0, 0)
-LASER_SIZE = 0.02
-
-# Capsule graphics
-CAPSULE_COLOR = graphicsUtils.formatColor(1, 1, 1)
-CAPSULE_SIZE = 0.25
-
-# Drawing walls
-WALL_RADIUS = 0.15
-
-class InfoPane:
+class CaptureInfoPane(AbstractPane):
     def __init__(self, layout, gridSize, redTeam, blueTeam):
-        self.gridSize = gridSize
-        self.width = (layout.width) * gridSize
-        self.base = (layout.height + 1) * gridSize
-        self.height = INFO_PANE_HEIGHT
-        self.fontSize = 24
-        self.textColor = PACMAN_COLOR
         self.redTeam = redTeam
         self.blueTeam = blueTeam
+        super().__init__(layout, gridSize)
         self.drawPane()
 
-    def toScreen(self, pos, y = None):
-        """
-        Translates a point relative from the bottom left of the info pane.
-        """
-
-        if (y is None):
-            x, y = pos
-        else:
-            x = pos
-
-        x = self.gridSize + x  # Margin
-        y = self.base + y
-        return x, y
-
     def drawPane(self):
         self.scoreText = graphicsUtils.text(self.toScreen(0, 0), self.textColor,
-                self._infoString(0, 1200), "Consolas", self.fontSize, "bold")
-        self.redText = graphicsUtils.text(self.toScreen(230, 0), TEAM_COLORS[0],
-                self._redScoreString(), "Consolas", self.fontSize, "bold")
-        self.redText = graphicsUtils.text(self.toScreen(690, 0), TEAM_COLORS[1],
-                self._blueScoreString(), "Consolas", self.fontSize, "bold")
+                self._infoString(0, 1200), graphicsConstants.CONSOLAS_FONT, self.fontSize,
```

Comment:
Lets just do all Times instead of consolas. Maybe change constant to default_font so we can change the font without having to change the constant